test/spinxsk/spinxsk.c

```diff
@@ -2589,6 +2953,7 @@ main(
 {
     HANDLE adminThread;
     HANDLE watchdogThread;
+    HANDLE globalConcurrentThreads[DEFAULT_GLOBAL_CONCURRENT_WORKERS_COUNT] = {0};
```

Comment:
maybe i'm missing something, but where does the `DEFAULT_GLOBAL_CONCURRENT_WORKERS_COUNT` get validated vs the input parameter?

IMO we should just alloc an array dynamically, solving both the validation problem and the ability to scale up to a 512-CPU machine without recompiling

Reply:
... worse than that actually. I forgot half the logic or something? Fixing that.